gdrivefs/gdtool/drive.py

```diff
@@ -770,10 +770,15 @@ def remove_entry(self, normalized_entry):
 
         client = self.__auth.get_client()
 
-        args = { 'fileId': normalized_entry.id }
+        args = {
```

Comment:
But then I have to check for delete_to_trash before every call to remove_entry. Shouldn't we then have something like a delegate function that either calls remove_entry or trash_entry based on the variable's value?